docs/source/basic/intro.rst

```diff
@@ -0,0 +1,90 @@
+Introduction
+============
+
+The *alpaka* library defines and implements an abstract interface for the *hierarchical redundant parallelism* model.
+This model exploits task- and data-parallelism as well as memory hierarchies at all levels of current multi-core architectures.
+This allows to achieve portability of performant codes across various types of accelerators by ignoring specific unsupported levels and utilizing only the ones supported on a specific accelerator.
```

Comment:
This question is not for this fragment per se, but the effect is also visible here. In readthedocs it looks visually quite weirdly with the left-side alignment. Do they support alignment by width?

Reply:
I add some css code, to justify the text bodies. You can check the result, if I pushed the commit.